content/release_notes.md

```diff
@@ -218,7 +262,7 @@ Use the `windows_user_privilege` resource to add users and groups to the specifi
 
 The `compile_time` property is now available for all resources so that they can be set to run at compile time without the need forcing the action.
```

Comment:
```suggestion
The `compile_time` property is now available for all resources so that they can be set to run at compile time without the need to force the action.
```